Frontend/ui-customer/src/Screens/CustomerUtils/AddNewAddressScreen/Controller.purs

```diff
@@ -303,8 +308,18 @@ getLocation prediction = {
   , alpha : 1.0
   , fullAddress : dummyAddress
   , locationItemType : Just PREDICTION
+  , distance : getFormattedDistance (fromMaybe 0 (prediction ^. _distance))
+  , showDistance : checkShowDistance (fromMaybe 0 (prediction ^. _distance))
 }
 
+getFormattedDistance :: Int -> String
+getFormattedDistance distanceInMeters
+  | distanceInMeters >= 1000 = parseFloat (toNumber distanceInMeters / 1000.0) 1 <> " km"
+  | otherwise = show distanceInMeters <> " m"
+
+checkShowDistance :: Int ->  Boolean
+checkShowDistance distance = if (distance > 0 && distance <= 50000) then true else false
```

Comment:
@prashant601 instead of returning value depending on a boolean value return the boolean value (condition) itself.